### DISCUSSION TITLE
Ignore baseUrl for some images, perhaps ?

### DISCUSSION THREAD

**Discussion by ffixitt** [maizzle/framework#960] [Q&A]
- Maizzle Version: v4.4.2
- Node.js Version: v14.21.2

I have content images what uploaded on CDN , but i have and images what generated from dashboard ( ex. avatars ). Is there any way to ignore some `<img>` as not to fill src from the baseURL ?

```js
// config.js
module.exports = {
   baseURL: {
        url: 'https://example.com/images/',
        tags: ['img', 'source', 'background'],
    },
}
```
```js
person: {
    avatar: "https://somedomain.com/img/user233224.png",
    name: "John Doe"
}
```

Input:
```html
<img src="@{{ person.avatar }}" alt="@{{ person.name }}">
```

Output:
```html
<img src="https://example.com/images/{{ person.avatar }}" alt="{{ person.name }}">
```

Expectation:
```html
<img src="{{ person.avatar }}" alt="{{ person.name }}">
```

**Comment by cossssmin** [comment]
Have a look at the baseURL docs, you can only specify at best what attribute values should be prepended what URLs:

https://maizzle.com/docs/transformers/base-url#attributes

You could use it to append only to a custom attribute and then rewrite that attribute name with `replaceStrings`:

https://maizzle.com/docs/transformers/replace-strings

**Comment by ffixitt** [comment]
@cossssmin  Okay, thx 🙏.

Maybe in the future it will be possible to add an attribute `ignore`, like a feature to ignore the compile image src.
